lib/Dialect/FIRRTL/Transforms/IMDeadCodeElim.cpp

```diff
@@ -524,8 +634,8 @@ void IMDeadCodeElimPass::rewriteModuleSignature(FModuleOp module) {
       if (argument.getType().isa<RefType>())
         continue;
 
-      // Ok, this port is used only within its defined module. So we can replace
```

Comment:
Seems like the auto-formatter disagreed with some comment line breaking. Was it wrong before?

Reply:
Hmm, I think I accidentally relayout in a different way, will fix. Thanks!